modules/distribution/src/repository/resources/dbscripts/apimgt/mssql.sql

```diff
@@ -338,73 +340,73 @@ ALTER TABLE SP_PROVISIONING_CONNECTOR ADD CONSTRAINT PRO_CONNECTOR_APPID_CONSTRA
 
 IF NOT  EXISTS (SELECT * FROM SYS.OBJECTS WHERE OBJECT_ID = OBJECT_ID(N'[DBO].[IDP]') AND TYPE IN (N'U'))
 CREATE TABLE IDP (
-            ID INTEGER IDENTITY,
-            TENANT_ID INTEGER,
-            NAME VARCHAR(254) NOT NULL,
-            IS_ENABLED CHAR(1) NOT NULL DEFAULT '1',
-            IS_PRIMARY CHAR(1) NOT NULL DEFAULT '0',
-            HOME_REALM_ID VARCHAR(254),
-            IMAGE VARBINARY(MAX),
-            CERTIFICATE VARBINARY(MAX),
-            ALIAS VARCHAR(254),
-            INBOUND_PROV_ENABLED CHAR (1) NOT NULL DEFAULT '0',
-            INBOUND_PROV_USER_STORE_ID VARCHAR(254),
-            USER_CLAIM_URI VARCHAR(254),
-            ROLE_CLAIM_URI VARCHAR(254),
-            DESCRIPTION VARCHAR (1024),
-            DEFAULT_AUTHENTICATOR_NAME VARCHAR(254),
-            DEFAULT_PRO_CONNECTOR_NAME VARCHAR(254),
-            PROVISIONING_ROLE VARCHAR(128),
-            IS_FEDERATION_HUB CHAR(1) NOT NULL DEFAULT '0',
-            IS_LOCAL_CLAIM_DIALECT CHAR(1) NOT NULL DEFAULT '0',
-            PRIMARY KEY (ID),
-            DISPLAY_NAME VARCHAR(255),
-            UNIQUE (TENANT_ID, NAME));
+			ID INTEGER IDENTITY,
+			TENANT_ID INTEGER,
+			NAME VARCHAR(254) NOT NULL,
+			IS_ENABLED CHAR(1) NOT NULL DEFAULT '1',
+			IS_PRIMARY CHAR(1) NOT NULL DEFAULT '0',
+			HOME_REALM_ID VARCHAR(254),
+			IMAGE VARBINARY(MAX),
+			CERTIFICATE VARBINARY(MAX),
+			ALIAS VARCHAR(254),
+			INBOUND_PROV_ENABLED CHAR (1) NOT NULL DEFAULT '0',
+			INBOUND_PROV_USER_STORE_ID VARCHAR(254),
+ 			USER_CLAIM_URI VARCHAR(254),
+ 			ROLE_CLAIM_URI VARCHAR(254),
+  			DESCRIPTION VARCHAR (1024),
+ 			DEFAULT_AUTHENTICATOR_NAME VARCHAR(254),
+ 			DEFAULT_PRO_CONNECTOR_NAME VARCHAR(254),
+ 			PROVISIONING_ROLE VARCHAR(128),
+ 			IS_FEDERATION_HUB CHAR(1) NOT NULL DEFAULT '0',
+ 			IS_LOCAL_CLAIM_DIALECT CHAR(1) NOT NULL DEFAULT '0',
+			PRIMARY KEY (ID),
+	        DISPLAY_NAME VARCHAR(255),
+			UNIQUE (TENANT_ID, NAME));
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04